content/blog/hackathon-2022-wazo-matrix-bridge.md

```diff
@@ -0,0 +1,60 @@
+Title: Hackathon 2022 - Bridging Wazo and Matrix
+Date: 2022-10-20 12:00:00
+Author: Jesse Sopel
+Category: Hackathon
+Tags: Chat, Python, Matrix, Bridge, Hackathon
+Slug: hackathon-2022-bridging-wazo-and-matrix
+Status: published
+
+# Hackathon 2022 - Bridging Wazo and Matrix
+
+Every year Wazo organizes a Hackathon with their employees, and as many of us as possible get together to work on some fun projects and see what we can learn.
+There were multiple teams working on different projects. Our team consisted of Charles, Francis, Francois and I (Jesse). 
+We decided to both work on creating a bridge between Wazo's built-in chat system and a Matrix server, as well as [testing out adding some new features to the existing chat](https://wazo-platform.org/blog/hackaton-2022-building-group-chat-with-wazo), but this article is focused on the former. 
```

Comment:
To keep in mind, if the other article is not yet ready, maybe the link should added in another PR

Reply:
Good point. I assumed they'd be released at the same time, but I'm in no rush to publish it. We can wait until yours is published if you want too.